ci/tests/e2e-helm.sh

```diff
@@ -159,20 +162,32 @@ test_operator() {
 # switch to the "default" namespace
 oc project default
 
-# create and build the operator
-pushd "$GOTMP"
-operator-sdk new nginx-operator --api-version=helm.example.com/v1alpha1 --kind=Nginx --type=helm
-
-pushd nginx-operator
-cp deploy/operator.yaml deploy/operator-copy.yaml
-sed -i "s|REPLACE_IMAGE|$IMAGE|g" deploy/operator.yaml
+# # create and build the operator
+# pushd "$GOTMP"
+# operator-sdk new nginx-operator --api-version=helm.example.com/v1alpha1 --kind=Nginx --type=helm
+#
+# pushd nginx-operator
+# cp deploy/operator.yaml deploy/operator-copy.yaml
+# sed -i "s|REPLACE_IMAGE|$IMAGE|g" deploy/operator.yaml
 
 OPERATORDIR="$(pwd)"
 
-trap_add 'remove_operator' EXIT
-deploy_operator
+# use sample in testdata
+pushd $ROOTDIR/testdata/helm/memcached-operator
+ls
+
+# trap_add 'remove_operator' EXIT
+
+# deploy_operator
+echo "running make deploy"
+make deploy IMG=$IMAGE
+
+echo "running test_operator"
 test_operator
-remove_operator
+
+# remove_operator
+echo "running make undeploy"
+trap_add 'make undeploy' EXIT
 
 # the nginx-operator pods remain after the deployment is gone; wait until the pods are removed
 if ! timeout 60s bash -c -- "until kubectl get pods -l name=nginx-operator |& grep \"No resources found\"; do sleep 2; done";
```

Comment:
the testdata sample is no longer a ngnix. It is a Memcached operator .  So,we need to change the checks.